internal/service/networkpeering/resource_network_peering_test.go

```diff
@@ -275,7 +319,7 @@ func configAzure(projectID, providerName, directoryID, subscriptionID, resourceG
 			azure_directory_id    = "%[3]s"
 			azure_subscription_id = "%[4]s"
 			resource_group_name   = "%[5]s"
-			vnet_name	            = "%[6]s"
+			vnet_name	          = "%[6]s"
```

Comment:
better %[6]q than "%[6]s", same for others

Reply:
done!